### ISSUE TITLE
Error while deploying with CircleCI 2.0

### ISSUE THREAD

**Issue by majorgilles** [coveralls-clients/coveralls-python#213]
Since a few weeks, I'm having issues deploying because coveralls fail with the following stack trace:

```
Traceback (most recent call last):
  File "/home/circleci/some-project/.venv/bin/coveralls", line 5, in <module>
    from coveralls.cli import main
  File "/home/circleci/some-project/.venv/lib/python3.6/site-packages/coveralls/__init__.py", line 2, in <module>
    from .api import Coveralls
  File "/home/circleci/some-project/.venv/lib/python3.6/site-packages/coveralls/api.py", line 13, in <module>
    from .reporter import CoverallReporter
  File "/home/circleci/some-project/.venv/lib/python3.6/site-packages/coveralls/reporter.py", line 10, in <module>
    from coverage.report import Reporter
ImportError: cannot import name 'Reporter'
```

I assume it is a specific dependency of coveralls that has recently changed?

**Comment by TheKevJames** [comment]
My best guess would be this is related to the recent coverage==5.0 release. We have an issue to fix this (#203) but I've been traveling a few weeks now and haven't had a chance to fix things up.

This `coveralls` library has a pin to `coverage<5`, but if you're installing coverage separately, you may be sidestepping this version pin -- could you try doing a `pip install coverage<5`?

**Comment by TheKevJames** [comment]
Closing as a dupe.
